src/CompScienceMeshes.jl

```diff
@@ -29,8 +29,10 @@ export weld
 # mesh refinement
 export barycentric_refinement, bisecting_refinement
 
+export  Loop_subdivision
+
 # submesh selection and cell retrieval
-export submesh
```

Comment:
One of your test still relies on this export. Could you make sure all your test run and commit the amendments?